x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/DataFrameField.java

```diff
@@ -27,6 +27,7 @@
     public static final ParseField SOURCE = new ParseField("source");
     public static final ParseField DESTINATION = new ParseField("dest");
     public static final ParseField FORCE = new ParseField("force");
+    public static final ParseField SIZE = new ParseField("size");
```

Comment:
I know `size` is what the search expects and in the context of a search it is a size. However, in datafeeds and now data frame transforms, this is better described as the `scroll_size` or `page_size`. I'm not feeling to strongly about this but it's worth considering.

Reply:
+1, rollup uses `page_size`, I find `size` to unspecific